### PR TITLE
verbosity level in command line & replace prints by loggings

### PR DESCRIPTION
**Pull request by AurelienJaquier**

### CODE REVIEW

File: emodelrunner/parsing_utilities.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    )
+    parser.add_argument("-v", "--verbose", action="count", dest="verbosity", default=0)
+    return parser.parse_args()
+
+
+def set_verbosity(verbosity):
+    """Set verbosity level.
+
+    Args:
+        verbosity (int): verbosity level. 0 for warning, 1 fro info and 2 or more for debug
```
Comment (anilbey) [inline emodelrunner/parsing_utilities.py:41]: small typo `fro -> for`

**Review by anilbey — APPROVED** [review]
Great work!
I just added one small comment.

**Review by anilbey — CHANGES_REQUESTED** [review]
Can you just add 2 unit tests one for `get_parser_args` and the other for `set_verbosity` to test them?

File: emodelrunner/parsing_utilities.py (hunk 1 above)
Comment (AurelienJaquier) [reply emodelrunner/parsing_utilities.py:41]: done in next commit

**Comment by AurelienJaquier** [comment]
Unfortunately, I couldn't test the logging level, since pytest overwrites it with its own. So I am only adding a test for the argument parsing function.

**Comment by anilbey** [comment]
Ok good, for this scenario there is a very cool built-in of Python that gives us ability to replace the module with a mock object and make sure that object is called the correct way.
 https://docs.python.org/3/library/unittest.mock.html
Let me show one example on this PR.

**Comment by AurelienJaquier** [comment]
Great, thank you very much, the new test looks very nice. It's ok to merge on my side. What do you think ?

**Review by anilbey — APPROVED** [review]
Also good for me, go ahead :)

**Pull request merged**